Gemfile.lock

```diff
@@ -489,7 +489,7 @@ GEM
     ruby-progressbar (1.7.1)
     rubyntlm (0.4.0)
     rubypants (0.2.0)
-    rugged (0.21.2)
+    rugged (0.21.4)
```

Comment:
If new rugged version will cause errors we might want to revert this line change.